src/index.ts

```diff
@@ -218,7 +218,7 @@ function resolveLaravelPlugin(pluginConfig: Required<PluginConfig>): LaravelPlug
                                 server.config.logger.info(`  ${colors.green('➜')}  Using Herd certificate to secure Vite.`)
                             }
 
-                            if (resolvedConfig.server.https.key.startsWith(valetConfigPath())) {
+                            if (resolvedConfig.server.https.key.startsWith(valetMacConfigPath()) || resolvedConfig.server.https.key.startsWith(valetLinuxConfigPath())) {
                                 server.config.logger.info(`  ${colors.green('➜')}  Using Valet certificate to secure Vite.`)
                             }
```

Comment:
@jameshulse, I've just made this change to ensure the Linux version also sees the log message in the Vite server output. Could you confirm that is working as expected?

Reply:
I sure do:

![image](https://github.com/user-attachments/assets/bbdd24f1-edc5-4758-af0a-f15645b322ce)

Reply:
Thanks!